monitoring/defender/autotask/on_module_factory_events.js

```diff
@@ -973,17 +971,15 @@ const checkIfIsOwner = async (provider, ensName, address, chainId) => {
 
 /**
  *
- * @param {Array} logs //Array of transactions logs
- * @param {*} utils //EtherJS utils
+ * @param {Array} logs //Array of transaction logs
+ * @param {string} realityMastercopy //RealityMasterCopy Address
+ * @param {*} utils
  * @returns
  */
-const decode = async (logs, utils) => {
+const decode = async (logs, realityMastercopy, utils) => {
```

Comment:
This is a very generic name. Could we change it to something more descriptive? For instance, `extractModuleProxyCreations`.

We should also check that the logs are emitted from our ProxyFactory contract.

Reply:
I cant see that you have addressed any of this concerns